src/Behat/Behat/Transformation/Call/Filter/DefinitionArgumentsTransformer.php

```diff
@@ -100,7 +100,11 @@ private function transformArgument(DefinitionCall $definitionCall, $index, $valu
                 continue;
             }
 
-            return $transformer->transformArgument($definitionCall, $index, $value);
+            $transformedValue = $transformer->transformArgument($definitionCall, $index, $value);
+
+            if ($value !== $transformedValue) {
```

Comment:
This condition is not needed. If they're equal you can still return `$transformedValue` (as `$value` is returned below);

Additionally the `continue` above doesn't add much to comprehension, you could flip that `if` and nest the `return` inside it.